build/sdk/js/templates/README.mustache

```diff
@@ -18,6 +18,27 @@ You can use this SDK to search for media, get information about media and about
 - For the status of the API, see [API status](https://status.developers.shutterstock.com/).
 - For the source code, see https://github.com/shutterstock/public-api-javascript-sdk.
 
+## Contributing
+
+- This SDK is generated at [shutterstock/public-api-sdk-generator](https://github.com/shutterstock/public-api-sdk-generator), please make changes to SDK there.
+- Changes to tests can be made directly here.
+
+## Tests
+
+To Run the tests you will need to authenticate with Shutterstock,
+and set the `SSTK_API_KEY` environment variable. See [here](https://api-reference.shutterstock.com/#oauth).
```

Comment:
To run the tests, you must authenticate with the Shutterstock API, get a token, and put the token in the `SHUTTERSTOCK_API_TOKEN` environment variable. See [Authentication](https://api-reference.shutterstock.com/#authentication).